src/digitalmarketplace/components/option-select/option-select.yaml

```diff
@@ -0,0 +1,96 @@
+params:
+- name: title
+  type: string
+  required: true
+  description: The title of the option select
+- name: name
+  type: string
+  required: true
+  description: The name of the option select element
+- name: options_container_id
+  type: string
+  required: true
+  description: The ID of the container for the Checkboxes component
+- name: items
+  type: array
+  required: true
+  description: The checkbox items
+- name: closed_on_load
+  type: boolean
+  description: Whether to collapse the filter by default
```

Comment:
If this is optional, what is the default value?

Reply:
Good spot, popped a default into the docs